site/content/config.js

```diff
@@ -7,6 +7,11 @@ const config = {
     { href: '/docs/roadmap', name: 'Roadmap' },
     { href: '/docs/guides', name: 'Guides' },
   ],
+  socialImageUrl: '',
```

Comment:
Why are we adding blank values here? Surely these should be left out and we have defaults in siteConfig.js?